src/components/ChartFactory.js

```diff
@@ -197,7 +199,10 @@ export default function ChartFactory({
             key={key}
             height={computedHeight}
             horizontal={horizontal}
-            labels={datum => numberFormatter.format(datum.y)}
+            labels={datum =>
+              numberFormatter.format(datum.y) +
```

Comment:
Isn't better to encapsulate `numberFormatter` behind a function that takes `value, (optional) unit` that does the formatting and appending unit rather than doing it for `labels`?

Reply:
Yeah, its always better to change on point @kilemensi